backend/alembic/README.md

```diff
@@ -28,9 +28,6 @@ Migrations should be created on all database changes using
 alembic revision -m "descriptive message here"
 ```
 
-This will generate a new file called something like "abcdef123456_descriptive_message_here.py". It will contain an "upgrade" method should take the
-current table state and update it to the new table schema, and a "downgrade" method that should take the upgraded table state and turn it back
-into the prior schema. It's recommended to use raw SQL for these migrations, as alembic does not support certain functionality (i.e cascade deletes)
-well.
+This will generate a new file called something like "abcdef123456_descriptive_message_here.py". It will contain an "upgrade" method should take the current table state and update it to the new table schema, and a "downgrade" method that should take the upgraded table state and turn it back into the prior schema well. It's recommended to use the sqlalchemy ORM to create the tables, however if you need to drop a table it's reccommended to use raw SQL since SQLAlchemy does not support `CASCADE` well.
```

Comment:
What does "since SQLAlchemy does not support `CASCADE` well." mean? In what does it not support `CASCADE` well?